geoextent/lib/helpfunctions.py

```diff
@@ -0,0 +1,90 @@
+
+import sys, os, platform, datetime, math 
+import getopt
+from osgeo import ogr
+from osgeo import osr
+from pyproj import Proj, transform
+
+WGS84_EPSG_ID = 4326
+
+def printObject(object):
```

Comment:
Please remove this function, and instead add nice printing to the classes we define ourselves, see https://stackoverflow.com/questions/1535327/how-to-print-instances-of-a-class-using-print#1535336

We never need to print, we should almost always use `logger`.

Reply:
Function removed and class created.